src/entries/popup/pages/walletSwitcher/renameWalletPrompt.tsx

```diff
@@ -53,6 +62,7 @@ const RenameWallet = ({
             variant="transparent"
             textAlign="center"
             autoFocus
```

Comment:
Any idea why this broke within the <Input> component? It works in a few other places currently IIRC

Reply:
Looks like that prop actually isn't being passed to the element for some reason:
```
<input class="_28iyjl0 _28iyjl4 _28iyjl5 _28iyjla _28iyjlgo _28iyjlhv _28iyjlj2 _28iyjlfh _28iyjlpm _28iyjl7l _28iyjl78 _28iyjlk8 _10vj57r0 _10vj57r4 _28iyjl1c2 _28iyjl1d8 _28iyjl1di _28iyjl1ct _28iyjl1dm _10vj57r6 _10vj57r7" data-testid="wallet-name-input" tabindex="1" autocomplete="off" placeholder="ex: Wallet 1" value="">
```